src/org/zaproxy/zap/extension/pscanrulesBeta/CSRFCountermeasures.java

```diff
@@ -215,46 +215,46 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 					// Give the first FORM tag as evidence
 					evidence = formElement.getFirstElement().getStartTag().toString();
```

Comment:
- The above if statement can be removed, it's always true;
 - The `evidence` can now be declared here.